api/http/handler/webhooks/handler.go

```diff
@@ -0,0 +1,42 @@
+package webhooks
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+	portainer "github.com/portainer/portainer"
+	httperror "github.com/portainer/portainer/http/error"
+	"github.com/portainer/portainer/http/proxy"
+	"github.com/portainer/portainer/http/security"
+)
+
+// Handler is the HTTP handler used to handle webhook operations.
+type Handler struct {
+	*mux.Router
+	WebhookService  portainer.WebhookService
+	ProxyManager    *proxy.Manager
+	EndpointService portainer.EndpointService
+	requestBouncer  *security.RequestBouncer
+}
+
+// NewHandler creates a handler to manage settings operations.
+func NewHandler(bouncer *security.RequestBouncer) *Handler {
+	h := &Handler{
+		Router:         mux.NewRouter(),
+		requestBouncer: bouncer,
+	}
+	// h.Handle("/webhooks",
+	// 	bouncer.RestrictedAccess(httperror.LoggerHandler(h.webhookCreate))).Methods(http.MethodPost)
+	// h.Handle("/webhook/{id}",
+	// 	bouncer.RestrictedAccess(httperror.LoggerHandler(h.webhookInspect))).Methods(http.MethodGet)
+
+	//Setting all handlers to public access for quick testing
+	h.Handle("/webhooks",
+		bouncer.PublicAccess(httperror.LoggerHandler(h.webhookCreate))).Methods(http.MethodPost)
```

Comment:
I believe that we should use `bouncer.AuthenticatedAccess`  instead of `bouncer.PublicAccess` for each HTTP endpoint here.